src/themes/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { StyleSheet } from 'react-native'
+
+const styleUtils = StyleSheet.create( {
```

Comment:
@Harjot1Singh instead of using `StyleSheet.create()` we could just do an object? Similar to the article you shared https://thoughtbot.com/blog/structure-for-styling-in-react-native